llvm/test/CodeGen/AIE/aie2/schedule/vextract.mir

```diff
@@ -114,3 +114,76 @@ body:             |
     $r3 = VEXTRACT_S16 $x3, $r16
     $r4 = VEXTRACT_S8  $x3, $r16
 ...
+
+---
```

Comment:
Here we show the absence of port conflicts, can we also have some tests showing conflicts?